packages/@vue/babel-preset-app/index.js

```diff
@@ -17,7 +17,8 @@ module.exports = (context, options = {}) => {
   const envOptions = {
     modules: options.modules || false,
     targets: options.targets,
-    useBuiltIns: typeof options.useBuiltIns === 'undefined' ? 'usage' : options.useBuiltIns
+    useBuiltIns: typeof options.useBuiltIns === 'undefined' ? 'usage' : options.useBuiltIns,
+    decoratorsLegacy: options.decoratorsLegacy || false
```

Comment:
Is this needed for `preset-env`? 
It seems `decoratorsLegacy` only affects stage-0/1/2.

Reply:
yeah it's only for stage 0,1,2. It's one of the unfortunate issues related to having stage presets at all since you might not even use decorators. This is why after a lot of discussion may just want to remove them since it's just an array of plugins that will constantly changing https://github.com/babel/babel/issues/7770.